NOTICE.md

```diff
@@ -0,0 +1,9 @@
+# Notice for Genomic Data Infrastructure 
+
+    This product was developed by Luxembourg National Data Service (LNDS), Center for Genomic Regulation and Health-RI.
```

Comment:
the indentation does not seems to be proper. can you check this? it should all start from left